### DISCUSSION TITLE
ansible-lint fqcn for local modules help

### DISCUSSION THREAD

**Discussion by jpatallah** [ansible/ansible-lint#3321] [Q&A]
Hello,

When adding ansible-lint to one of my ansible roles it's complaining that i'm not using the fqcn for some of my tasks.  We have local modules we've created.

For example:
- name: Create/remove/modify AD users
  win_adusers_cmodule:

is located in library/win_adusers_cmodule.py

error:
fqcn[action]: Use FQCN for module actions, such `<namespace>.<collection>.win_adusers_cmodule`.
tasks/main.yml:2 Action `win_adusers_cmodule` is not FQCN.

What is the format for the fqcn for locally referenced modules?

**Comment by MarkusTeufelberger** [comment]
I assume it could be `ansible.legacy`? https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#what-is-the-difference-between-ansible-legacy-and-ansible-builtin-collections
